services/GraphQL/queries/providers.ts

```diff
@@ -0,0 +1,46 @@
+import { gql } from '@apollo/client'
+
+export const GET_PROVIDERS = gql`
+  query {
+    providers {
+      id
+      commercial_name
+      sales_email
+      sales_phone
+      address
+      created_at
+      updated_at
+      avatar_url
+    }
+  }
+`
+
+// get providers per categories
+export const GET_PROVIDERS_BY_CATEGORY = gql`
+  query ($category_id: String!) {
+    categories(where: { id: { _eq: $category_id } }) {
```

Comment:
Here you're using where, but you could to use:

```suggestion
    categories(id: $category_id) {
```
?

Reply:
I'm receiving errors from this query
![lastscreenshot](https://user-images.githubusercontent.com/66505715/166562101-95a9f420-8be9-4eba-b8ca-d927b37e8851.png)

Reply:
![lastscreenshot](https://user-images.githubusercontent.com/66505715/166562208-b52717c4-a360-4e26-a9ef-895869e0f7e0.png)